tests/gtest/avifsvttest.cc

```diff
@@ -1,12 +1,41 @@
 // Copyright 2025 Google LLC
 // SPDX-License-Identifier: BSD-2-Clause
 
+#include <tuple>
 #include <vector>
 
 #include "avif/avif.h"
 #include "aviftest_helpers.h"
 #include "gtest/gtest.h"
 
+#if defined(__has_include)
+#if __has_include("svt-av1/EbSvtAv1Enc.h")
+#include "svt-av1/EbSvtAv1Enc.h"
+
+// Copied from src/codec_svt.c.
+#ifndef SVT_AV1_VERSION_MAJOR
+#define SVT_AV1_VERSION_MAJOR SVT_VERSION_MAJOR
+#define SVT_AV1_VERSION_MINOR SVT_VERSION_MINOR
+#define SVT_AV1_VERSION_PATCHLEVEL SVT_VERSION_PATCHLEVEL
+#define SVT_AV1_CHECK_VERSION(major, minor, patch)                          \
+  (SVT_AV1_VERSION_MAJOR > (major) ||                                       \
+   (SVT_AV1_VERSION_MAJOR == (major) && SVT_AV1_VERSION_MINOR > (minor)) || \
+   (SVT_AV1_VERSION_MAJOR == (major) && SVT_AV1_VERSION_MINOR == (minor) && \
+    SVT_AV1_VERSION_PATCHLEVEL >= (patch)))
+#endif
+
+#if SVT_AV1_CHECK_VERSION(3, 0, 0)
+constexpr bool kSvtAv1SupportsLossless = true;
+#else
+constexpr bool kSvtAv1SupportsLossless = false;
+#endif
+#else
+constexpr bool kSvtAv1SupportsLossless = false;
+#endif
+#else
+constexpr bool kSvtAv1SupportsLossless = false;
+#endif
```

Comment:
Optional: If we build this file (avifsvttest.cc) only when SVT-AV1 is used, then we don't need to check `__has_include("svt-av1/EbSvtAv1Enc.h")`. That will get rid of the two `#if`'s at lines 11-12.

Reply:
Done.